admin/app/components/solidus_admin/users/edit/component.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+class SolidusAdmin::Users::Edit::Component < SolidusAdmin::BaseComponent
+  include SolidusAdmin::Layout::PageHelpers
+
+  def initialize(user:)
+    @user = user
+  end
+
+  def form_id
+    @form_id ||= "#{stimulus_id}--form-#{@user.id}"
+  end
+
+  def tabs
+    [
+      {
+        text: t('.account'),
+        href: solidus_admin.users_path,
+        current: action_name == "show",
+      },
+      {
+        text: t('.addresses'),
+        href: spree.addresses_admin_user_path(@user),
+        # @todo: update this "current" logic once folded into new admin
+        current: action_name != "show",
+      },
+      {
+        text: t('.order_history'),
+        href: spree.orders_admin_user_path(@user),
+        # @todo: update this "current" logic once folded into new admin
+        current: action_name != "show",
+      },
+      {
+        text: t('.items'),
+        href: spree.items_admin_user_path(@user),
+        # @todo: update this "current" logic once folded into new admin
+        current: action_name != "show",
+      },
+      {
+        text: t('.store_credit'),
+        href: spree.admin_user_store_credits_path(@user),
+        # @todo: update this "current" logic once folded into new admin
+        current: action_name != "show",
+      },
+    ]
+  end
+
+  def last_login(user)
+    return t('.last_login.never') if user.try(:last_sign_in_at).blank?
+
+    t(
+      '.last_login.login_time_ago',
+      # @note The second `.try` is only here for the specs to work.
```

Comment:
I think this is not only because of specs to pass. We allow custom user classes (besides the default `Spree::User` from `solidus_auth_devise`) and those user classes might not have this attribute.

Reply:
True, this comment is a bit misleading as the reason the specs require it is precisely because they use a non-`Spree::User` class. I can update it!

Reply:
(Specs use `Spree::LegacyUser` by default it seems)

Reply:
Yes, on purpose. I think updating the comment is ok

Reply:
Updated